src/blueprints/Do.Blueprints.Service.Application/Configuration/ConfigurationExtensions.cs

```diff
@@ -1,11 +1,62 @@
 using Do.Architecture;
 using Do.Configuration;
+using Do.Testing;
 using Microsoft.Extensions.Configuration;
+using Moq;
 
 namespace Do;
 
 public static class ConfigurationExtensions
 {
     public static void AddConfiguration(this List<ILayer> source) => source.Add(new ConfigurationLayer());
     public static void ConfigureConfigurationBuilder(this LayerConfigurator source, Action<IConfigurationBuilder> configuration) => source.Configure(configuration);
+
+    public static void ASetting<T>(this Mocker mockMe,
```

Comment:
my bad :raising_hand: 

these better belong to `TestingExtensions`, because configuration stub comes from there